### DISCUSSION TITLE
Is it possible to set an axis in log scale

### DISCUSSION THREAD

**Discussion by DylanDeraed** [cetz-package/cetz-plot#60] [Q&A] [answered]
Hello ! 
I am trying to write a report where I would like to plot some datas. I Would like to make a semi-log plot where only the x axis is in log scale. Is it possible ?

**Comment by johannes-wolf** [comment]
That is currently not built-in, no.

**Reply by tapyu** [reply]
:( Log and semi-log plots are rather important. Acc

**Reply by johannes-wolf** [reply]
I do not have the time right now to work on it :(.

**Comment by tapyu** [comment]
While we don't have the `linear`, `semilog`, or `log` options, you can enforce semilog scale by using something like this

```
#let transformed_data = data.map(x => (calc.ln(x.at(0)), x.at(1)))
```

**Comment by johannes-wolf** [comment]
Link: #568

**Comment by johannes-wolf** [comment, marked as answer]
With cetz-plot 0.1.0, log-scaling is now available by using `x-mode: "log"` or `y-mode: "log"`.
